WhatsInSeason/DESCRIPTION

```diff
@@ -0,0 +1,12 @@
+Package: WhatsInSeason
+Type: Package
+Title: Tells You the Seasonal Fruit & Veg in Your Region
+Version: 0.1.0
+Author: Annie Johansson
+Maintainer: Annie Johansson  <annie.johansson@student.uva.nl>
+Description: One step to becoming more environmentally friendly is to consume local produce that is in season. 
+    This package informs about what fruits and vegatables are in season in different countries around Europe. 
```

Comment:
Sounds like a really useful package, and the description is clear and informative.